source/lp_ticker.c

```diff
@@ -50,20 +53,33 @@ void lp_ticker_init(void) {
          * We want to use faster speeds, so we use 0.5ms tick for a max. of ~131MHz.
          * MAKE SURE `MINAR_PLATFORM_TIME_BASE` in the target configuration is set to `2000`!
          */
-        // Get clock configuration
-        RCC_ClkInitTypeDef RCC_ClkInitStruct;
-        uint32_t PclkFreq;
-        // Note: PclkFreq contains here the Latency (not used after)
-        HAL_RCC_GetClockConfig(&RCC_ClkInitStruct, &PclkFreq);
-        // Get TIM1 clock value
-        PclkFreq = HAL_RCC_GetPCLK1Freq();
-        // TIMxCLK = PCLKx when the APB prescaler = 1 else TIMxCLK = 2 * PCLKx
-        if (RCC_ClkInitStruct.APB1CLKDivider != RCC_HCLK_DIV1) {
-            PclkFreq *= 2;
-        }
-        TimMasterHandle.Instance               = TIM2;
+	// Get clock configuration
+	HAL_RCC_GetClockConfig(&RCC_ClkInitStruct, &TimClk);
```

Comment:
Anyway, I am not really concerned which solution to prefer. 
The only thing which I think is important is to comment this very carefully regarding both ```MINAR_PLATFORM_TIME_BASE```, the assumption made upon max ```pclk``` speeds, and the 16-bit prescaler restriction. Future developers will appreciate!